src/main/java/uk/gov/hmcts/darts/transcriptions/controller/TranscriptionController.java

```diff
@@ -62,7 +62,7 @@ public class TranscriptionController implements TranscriptionApi {
     @Override
     @SecurityRequirement(name = SECURITY_SCHEMES_BEARER_AUTH)
     @Authorisation(bodyAuthorisation = true, contextId = ANY_ENTITY_ID,
-        securityRoles = {JUDGE, REQUESTER, APPROVER, TRANSCRIBER, TRANSLATION_QA, RCJ_APPEALS},
+        securityRoles = {JUDGE, REQUESTER, APPROVER, TRANSCRIBER, TRANSLATION_QA},
```

Comment:
@hemantasharma1129 I think we have too many roles here - requestTranscription should only have 
`securityRoles = {JUDGE, REQUESTER, APPROVER}`?

Looking at https://tools.hmcts.net/confluence/display/DMP/Authorisation+Service+LLD

Reply:
i fixed this particular one, but didn;t check other endpoints